deploy/docker/docker-entrypoint.sh

```diff
@@ -107,8 +107,10 @@ fi
 
 # echo value of $VAR_FULL_NAME hiding secrets if any
 echo_value () {
+    # get MASK_CONFIG
+    MASK_CONFIG_FILTER=${MASK_CONFIG_FILTER:-password|passwd|key|token|secret}
```

Comment:
```suggestion
    MASK_CONFIG_FILTER="$MASK_CONFIG_FILTER|password|passwd|key|token|secret"
    FORMAT_MASK_CONFIG_FILTER=$(echo $MASK_CONFIG_FILTER |sed -e "s/^[^A-Za-z0-9_]\{1,\}//g"|sed -e "s/[^A-Za-z0-9_]\{1,\}/\|/g")

```

Reply:
Users can use any comma, space, or other separators to split `MASK_CONFIG_FILTER`

The `MASK_CONFIG_FILTER` set by the user will not override the default value, they will exist at the same time